tests/e2e/actions.go

```diff
@@ -717,9 +737,17 @@ func (tr TestConfig) addChainToGorelayer(
 	ChainId := tr.chainConfigs[action.Chain].ChainId
 	rpcAddr := "http://" + queryNodeIP + ":26658"
 
+	var accountPrefix string
+	if ChainId == ChainID("provi") {
```

Comment:
wdyt about having this in the ChainConfig https://github.com/cosmos/interchain-security/blob/f0b9d7b3b6139181687fcada146a74e88fcec18e/tests/e2e/config.go#L76 instead of in this "ad-hoc" way?

i.e. have a field in the chainConfig called "AccountPrefix", then here calling
```
accountPrefix := tr.chainConfigs[action.Chain].AccountPrefix
```

or something like that

Reply:
Good idea. See https://github.com/cosmos/interchain-security/pull/1570/commits/7a8a5d55cc9f09df83c0e34a2ca0ba1a899b52af